depthmapX/views/plotview/plotview.cpp

```diff
@@ -269,15 +272,36 @@ bool QPlotView::Output(QPainter *pDC, QGraphDoc *pDoc, bool screendraw)
       return false;
    }
 
-   AttributeTable& table = pDoc->m_meta_graph->getAttributeTable();
+   dXreimpl::AttributeTable& table = pDoc->m_meta_graph->getAttributeTable();
+   AttributeTableHandle& tableHandle = pDoc->m_meta_graph->getAttributeTableHandle();
+   LayerManagerImpl& layers = pDoc->m_meta_graph->getLayers();
    int displaycol = pDoc->m_meta_graph->getDisplayedAttribute();
 
    QRect rect = QRect(0, 0, width(), height());
    int mindim = __min(rect.width(),rect.height());
-   int viscount = table.getVisibleRowCount();
-   if(!viscount) m_x_axis = m_y_axis = 0;
 
-   int spacer = floor(2.0 * sqrt(double(mindim)/double(viscount != 0 ? viscount : 1)));
+   // do all the calculations here for the moment

```

Comment:
Hmm, that looks like something that should be in a helper class/method

Reply:
It should be indeed, but I tried looking into this and the proper solution, where these stats would be stored in the attribute table like the other stats, and it's quite complex to handle in this PR, due to having to update them every time the layers change. Thus, I have just updated the comment to fix at a later stage.